vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-client/src/main/java/com/vaadin/addon/spreadsheet/client/SpreadsheetWidget.java

```diff
@@ -220,6 +226,24 @@ public void onAttachOrDetach(AttachEvent attachEvent) {
 
             }
         });
+
+        if (isTouchMode() && !BrowserInfo.get().isAndroid()) {
```

Comment:
Out of curiosity, why are we excluding Android?

Reply:
https://caniuse.com/mdn-api_element_contextmenu_event

iOS is the only mobile platform that doesn't support the `contextmenu` event.

Reply:
This is correct, but after this comment, I checked and the context menu wasn't showing up on long press in Android devices. That was because, although `contextmenu` is supported by Chrome in Android, the Spreadsheet client wasn't listening to any event of that type.

I refactored it to now listen to `contexmenu` event instead of checking for right-click on `mouseup` event.